starters/saas/Cargo.toml

```diff
@@ -40,5 +40,5 @@ required-features = []
 [dev-dependencies]
 serial_test = "2.0.0"
 rstest = "0.18.2"
-loco-rs = { version = "0.2.3", features = ["testing"] }
+loco-rs = { version = "*", path = "../../", features = ["testing"] }
```

Comment:
When a user initializes a new project using our CLI, the starters are downloaded. However, when you point to path  all starters may break. This it why we set a static version version. 
This approach not only ensures stability for users' projects but also allows us to release new versions without immediate propagation to all users. It's worth noting that our deployment script automatically updates the specified version.